### PR TITLE
silence

### PR DESCRIPTION
**Pull request by grosser**
@bquorning 

screen shot 2017-03-07 at 4 18 49 pm

### CODE REVIEW

**Comment by grosser** [comment]
neat, updated!

**Comment by aud** [comment]
👋 Hey this is really spamming our logs. Is there anything I can do to help expedite things?

**Comment by grosser** [comment]
how about a +1 ? :D

**Comment by bquorning** [comment]
:+1:

**Comment by grosser** [comment]
v1.14.3

File: lib/zendesk_api/silent_mash.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require 'hashie'
+
+module ZendeskAPI
+  # @private
+  class SilentMash < Hashie::Mash
+    disable_warnings
```
Comment (pschambacher) [inline lib/zendesk_api/silent_mash.rb]: Would have been nice to check for method existence 😄

Comment (grosser) [inline lib/zendesk_api/silent_mash.rb]: update your hashie ... but I guess should bump required version here and yank the previous release ...

Comment (pschambacher) [inline lib/zendesk_api/silent_mash.rb]: Yeah I found out that I needed to bump hashie but it came as a bit of a surprise!

**Pull request merged**